utkdigitalinitiatives/utk_isl_xml_forms
# Add XML Form for ETDs, Pull Request Template, and Make README a markdown file.

**JIRA Ticket**: [TRAC-231](https://jira.lib.utk.edu/browse/TRAC-231)
# What Does This Do?
1. Adds a XML Form for UTK ETDs.
2. Adds a Pull Request Template.
3. Makes the README a markdown file.
   # What's new?

An XML Form for ETDS has been added. This will allow users to add a metadata record that meets our data dictionary. So far, only Create operations have been thoroughly tested.  Read and Update operations will be addressed in a future pull request.

A pull request template was added.

The README was transformed to markdown.
# How should this be tested?

Add the new form to form builder and associate with your choice of content model. Test it in form builder and press submit to see results.
# Additional notes:

I think this is safe to merge without testing.  There are two fields that I'm leaving for now, but I'll remove before we tag the release of TRACE Sprint 5.

Also, I hate that all of CRUD isn't working, but I thought it was important to get this in so I can remove blockers and we can get this into the hands of others.
# Interested parties:

@CanOfBees
